lib/open_food_network/i18n_inflections.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+module OpenFoodNetwork
+  # Provides efficient access to unit name inflections.
```

Comment:
this "unit name" should be replaced by "word" as this is no longer related specifically to unit_names. right?